packages/suite/src/components/suite/Settings/components/SettingsSection.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { ReactNode, ReactElement } from 'react';
+import styled from 'styled-components';
+import Card from '@suite-components/Card';
```

Comment:
I suppose the suite one can have a header... Otherwise they are identical. If it's with a header (which is outside the actual card as I understand) it's not really a card, more like a section. Also I really dig the Card props in `SettingsSection` :D

```
largePadding noVerticalPadding noPadding
```

Reply:
I can create an issue for removing the Card duplicate. For now I will leave it as is but rename the component itself.

Reply:
done

Reply:
good enough thanks